nginx/parts/common.conf.j2

```diff
@@ -40,10 +40,11 @@ location /{{ hconfigs['proxy_path_client'] }}/gh/ {
   sub_filter    src="/ src="/{{ hconfigs['proxy_path'] }}/gh/;
 }
 
-location /{{ hconfigs['proxy_path_client'] }}/dns/ {
+location /{{ hconfigs['proxy_path_client'] }}/dns/dns-query/ {
   # use it in your browser as https://defaultserverhost/{{ hconfigs['proxy_path_client'] }}/dns/dns-query{?dns}
-  set $upstream https://dns.google:443/; 
-  proxy_pass $upstream;
+  
+  #proxy_pass https://dns.google/dns-query;
+  proxy_pass https://1.1.1.1/dns-query;
```

Comment:
doh of 1.1.1.1 is limited to 10 request/s

doh of 8.8.8.8 is limited to 1500 r/s